bin/flutterflow_cli.dart

```diff
@@ -53,6 +54,15 @@ ArgResults _parseArgs(List<String> args) {
       negatable: true,
       help: 'Run "dart fix" on the downloaded code.',
       defaultsTo: false,
+    )
+    ..addFlag(
+      'parent-folder',
+      negatable: true,
+      help: 'Download into a sub-folder. By default, project is downloaded \n'
+          'into a folder named <project>. \n Setting this flag to false will '
```

Comment:
Nit: remove space between `\n` and `Settings`, otherwise `dart bin/flutterflow_cli.dart export-code --help` output looks a little weird.

Reply:
Done